pkg/cli/term/reader_windows.go

```diff
@@ -106,13 +107,67 @@ var keyCodeToRune = map[uint16]rune{
 // A subset of constants listed in
 // https://docs.microsoft.com/en-us/windows/console/key-event-record-str
 const (
-	leftAlt   = 0x02
-	leftCtrl  = 0x08
-	rightAlt  = 0x01
-	rightCtrl = 0x04
-	shift     = 0x10
+	leftAlt   uint32 = 0x02
+	leftCtrl  uint32 = 0x08
+	rightAlt  uint32 = 0x01
+	rightCtrl uint32 = 0x04
+	shift     uint32 = 0x10
 )
 
+type HKL (uint32)
```

Comment:
It doesn't seem like the parens are needed based on my understanding of Go and empirical testing on Windows 10 that shows it compiles and passes existing tests with the parens removed. Were they leftover from a previous version of this change or are they really needed?

Reply:
No, that's just because I'm new to Go. I'll remove them